app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HotelsController < ApplicationController
+  def show 
+    @hotels = Hotel.all
+    @hotel = Hotel.find(params[:id])
+  end
+end
```

Comment:
Why do we need all the hotels on the hotel show page? I don't think line 3 is necessary here.